index.html

```diff
@@ -1,260 +1,367 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
-    <title>David's Portfolio - Mobile</title>
-</head>
-<body>
-    <section class="container">
-        <section class="pg-1">
-            <header>
-    
-            </header>
-    
-            <div class="logo">
-                <h2 class="my-logo"><a href="#" target="_blank" rel="noopener noreferrer">Welcome</a></h2>
-                <nav>
-                    <img class="ham" src="pics/hamburger.png" alt="hamburger">
-                </nav>
-            </div>
-            <!-- Start of homepage section -->
-            <section class="description">
-    
-                <div class="name">
-                    <h1>I’m David <br> Glad to see <br> you!</h1>
-    
-                    <p>I’m a software developer! I can help you build a product , feature or website Look through some
-                        of my work and experience! If you like what you see and have a project you need coded, don’t
-                        hestiate to contact me. </p>
-    
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+        <title>David's Portfolio - Mobile</title>
+    </head>
+    <body>
+        <div>
+            <!-- start of desktop nav -->
+            <nav class="desktop-nav">
+                <div class="desktop-logo">
+                    <a href="#" target="_blank" rel="noopener noreferrer">
+                        Welcome
+                    </a>
                 </div>
```

Comment:
Please try to match the Figma design as much as possible at any point between `992px` and `1100px`
| Your design | Figma design |
| --- | --- |
| ![screencapture-raw-githack-davidobi-Portfolio-mobile-version-desktop-version-design-index-html-2022-02-17-14_26_56](https://user-images.githubusercontent.com/24703171/154482772-3c851de5-2f95-4942-97bd-5899493fc4ab.png) | ![Template 2-Main](https://user-images.githubusercontent.com/24703171/154482988-f39d07fc-89fe-48e3-a34d-16871ce3c3e8.png) |